src/proto/gps.proto

```diff
@@ -20,6 +20,9 @@ enum SampleType {
   TOTAL_DATA_TYPES = 13;
   POSITION = 14;
   LINEAR_VELOCITY = 15;
+  RGB_IMAGE_SIZE = 16;
+  DOMAIN_IMAGE = 17;
```

Comment:
@avivt Can you shed any light on what DOMAIN_IMAGE is doing? I couldn't tell by reading the code.